Mathlib/Algebra/CharZero/Defs.lean

```diff
@@ -100,21 +99,23 @@ theorem cast_ne_one {n : ℕ} : (n : R) ≠ 1 ↔ n ≠ 1 :=
   cast_eq_one.not
 #align nat.cast_ne_one Nat.cast_ne_one
 
+instance AtLeastTwo.instNeZero (n : ℕ) [n.AtLeastTwo] : NeZero n :=
+  ⟨Nat.ne_of_gt (Nat.le_of_lt one_lt)⟩
+
```

Comment:
I think I'd like to see this change, along with the other API changes around `AtLeastTwo`, split to its own PR. I don't think it's particularly controversial, but i'd rather it not be hidden amidst an otherwise-mechanistic patch.

Reply:
Makes sense! Done at #10964.